content/300-guides/200-deployment/110-deployment-guides/200-deploying-to-heroku.mdx

```diff
@@ -36,7 +36,7 @@ While the example uses REST, the same principles apply to a GraphQL server, with
 - Node.js installed.
 - PostgreSQL CLI `psql` installed.
 
-> **Note:** Heroku provides a free plan which includes Node.js hosting and a PostgreSQL database, so no billing information is required.
+> **Note:** Heroku no longer provides a free plan, so billing information is required.
```

Comment:
let's phrase this in a way that it won't be confusing for people who didn't know that Heroku offered a free plan before.

```suggestion
> **Note:** Heroku doesn't provide a free plan, so billing information is required.
```